README.md

```diff
@@ -69,19 +69,18 @@ handle different payloads and keys. Because of this, each record is individually
 
 ## Upsert Behavior
 
-If the target table already contains a record with the same key, the Destination will upsert with its current received
+If the target tables already contains a record with the same key, the Destination will upsert with its current received
```

Comment:
Need to be careful with that find / replace 😄 A few things were renamed that shouldn't have been.